services/streammanager/kafka/kafkamanager.go

```diff
@@ -594,11 +598,11 @@ func sendMessage(ctx context.Context, jsonData json.RawMessage, p producerManage
 	}
 
 	timestamp := time.Now()
-	userID, _ := parsedJSON.Get("userId").Value().(string)
+	userID := parsedJSON.Get("userId").String()
 	codecs := p.getCodecs()
 	if len(codecs) > 0 {
-		schemaId, _ := parsedJSON.Get("schemaId").Value().(string)
-		messageId, _ := parsedJSON.Get("message.messageId").Value().(string)
+		schemaId := parsedJSON.Get("schemaId").String()
+		messageId := parsedJSON.Get("message.messageId").String()
```

Comment:
what happens if parsedJSON.Get("schemaId") is not string or doesn't exist?

Reply:
It will be handled as  `""` if not present if its other type that is also handled here
https://github.com/tidwall/gjson/blob/8bf80a300b41c9f7c43406ee300606b217cf08c2/gjson.go#L72